modules/homebrew.nix

```diff
@@ -218,14 +219,27 @@ let
       # and error message with an assertion below if it's set by the user.
       noLock = mkOption { visible = false; default = null; };
 
+      analytics = lib.mkEnableOption ''
+        Enable Homebrew analytics.
+
+        See "https://docs.brew.sh/Analytics".  Setting this to `false` (default)
+        will turn the analytics off.  Setting this to `true` will turn them on.
+
+        Implementation note: when disabled, this option sets the
+        `HOMEBREW_NO_ANALYTICS` environment variable, by adding it to
+        [](#opt-environment.variables).
+
+      '';
+
```

Comment:
To match the other global options, can you change this to `mkOption` and set `type`, `default`, and `description` manually?